Add Fusion to dashboard (#710)
seqeralabs/docs: platform-cloud/docs/monitoring/dashboard.md

Select **Export data** in the view panel near the top of the page to export a CSV of the dashboard data for the selected organizations and workspaces.

## Fusion

> Select a workspace from the drop-down menu to view a Fusion usage comparison between the current month and the previous month. The usage value is displayed in GB with a percentage comparison against usage in the previous month).

Reviewer: ```suggestion
Select a workspace from the drop-down menu to view Fusion usage for the current and previous month. The usage is displayed in GB and shows a percentage change from the previous month.
```
